tests/fixtures/gtfs2ntfs/minimal/input/agency.txt

```diff
@@ -1,2 +1,2 @@
-agency_name,agency_url,agency_timezone
-mon agence,http://kisio.org,Europe/Paris
+agency_id,agency_name,agency_url,agency_timezone
+mon_agence,mon agence,http://kisio.org,Europe/Paris
```

Comment:
I don't see how you can define this fixture as **minimal** since it got multiple `calendar_dates`, multiple `stops`, multiple `trips`, etc. This definition of **minimal** doesn't make any sense and seems different for everyone. That said, I moved this file back to its original version since we can still test there is a new entry in `object_codes.txt` (see 046052d24624f0ecbb5091d43359c15d46027e33).

Reply:
minimal means just the minimum of data for reading a GTFS. no optional files. no extra columns..